components/rpill/browser/rpill.cc

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include <locale>
+#include <string>
+
+#include "brave/components/rpill/browser/rpill.h"
+
+#include "base/strings/string_util.h"
+#include "base/win/wmi.h"
+#include "content/public/browser/browser_thread.h"
+
+namespace rpill {
+
+bool exec() {
+  DCHECK_CURRENTLY_ON(content::BrowserThread::IO);
+  base::win::WmiComputerSystemInfo info =
+      base::win::WmiComputerSystemInfo::Get();
+  auto m = info.manufacturer();
+  auto mo = info.model();
+  const std::string man(m.begin(), m.end());
+  const std::string mod(mo.begin(), mo.end());
+  const std::string manmod = base::ToLowerASCII(man) + base::ToLowerASCII(mod);
+
+  return manmod.find("vm") != std::string::npos ||
```

Comment:
shouldn't these be `&&`?

Reply:
or better yet change `!=` to `==` and keep `||`?

Reply:
We want the result to be `true` if the string contains _at least_ one of those substrings. So `||` is more appropriate is this case.